src/cms/src/api/protected-area/content-types/protected-area/schema.json

```diff
@@ -373,6 +373,12 @@
       "relation": "oneToOne",
       "target": "api::park-gate.park-gate",
       "mappedBy": "protectedArea"
+    },
+    "parkFeatures": {
+      "type": "relation",
+      "relation": "oneToMany",
+      "target": "api::park-feature.park-feature",
+      "mappedBy": "protectedArea"
```

Comment:
We'd need to add `parkAreas` too. `protectedArea` should be able to have many `parkAreas`.

```json
"parkAreas": {
      "type": "relation",
      "relation": "oneToMany",
      "target": "api::park-area.park-area",
      "mappedBy": "protectedArea"
    },
```